letsdrill.rb

```diff
@@ -1,24 +1,32 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+if integer >= 90
+  return "A"
+elsif integer >= 80
+  return "B"
+elsif integer >= 70
+  return "C"
+elsif integer >=60
+  return "D"
+else
+  return "F"
+end
 end
 
 def shortest_string(array)
+array.sort_by {|x| x.length}.first
```

Comment:
Nice.  There's also `min_by`.  This might also be a good chance to use the following sort of syntax, which is shorthand for what you have here: `sort_by(&:length).first`